docs/cli.md

```diff
@@ -28,13 +28,13 @@ the same justification server for minting tokens, add the following to your
 `.bashrc` or `.zshrc` file:
 
 ```shell
-export JVSCTL_SERVER_ADDRESS="https://jvs.corp.internal:8080"
+export JVSCTL_SERVER_ADDRESS="jvs.corp.internal:443"
```

Comment:
8080 is the correct default: https://github.com/abcxyz/jvs/blob/61f3158c299259a73471b9b0b5616f06638b2573/pkg/config/justification_config.go#L36